sql/executor.go

```diff
@@ -313,13 +321,14 @@ func runExtendedSQL(slct string, db *DB, modelDir string, session *pb.Session) *
 				if os.Getenv("SQLFLOW_submitter") == "elasticdl" {
 					return elasticDLTrain(wr, pr, db, cwd, session, nil)
 				}
-				// TODO(weiguo): fix the hard code 0.8
-				ds, e := newTrainAndValDataset(db, pr.standardSelect.String(), pr.standardSelect.tables[0], 0.8)
-				if e != nil {
-					return e
+				var ds *trainAndValDataset
+				if !isUnsupervisedLearning(pr) {
```

Comment:
If we use `isUnsupervisedLearning` to skip `newTrainAndValDataset` process, how can we make sure the logic of checking  existing table with same name will be run when invoke `createDataset` in `newTrainAndValDataset` ?